docker-compose.yml

```diff
@@ -135,6 +135,7 @@ services:
       - USERID=${UID:-1001}
       - GROUPID=${GID:-1001}
     volumes:
+      - ./:/pecan
```

Comment:
I think we want to add the compiled pages to the `docs` image at build time rather than sharing them as a volume -- see my comment below. But if @robkooper contradicts me, listen to him instead :)